src/style.css

```diff
@@ -1,73 +1,126 @@
+/* ------------ youtube.com/CodeExplained ------------ */
 body {
-  background-color: rgb(241, 233, 233);
+  padding: 0;
+  margin: 0;
```

Comment:
- [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). ❌

I cannot make your task complete or incomplete.